package.json

```diff
@@ -31,7 +31,7 @@
     "properties-panel"
   ],
   "main": "dist/index.js",
-  "module": "dist/index.esm.js",
```

Comment:
Last time we tried we found this causes issues. Did our assessment change? 

I'd at least make this explicit via a separate commit.

Reply:
Hmm I thought it worked fine. I'll revert this change as I'm not 100% sure, and we have not adopted it in bpmn-js-properties-panel yet.

Reply:
If I remember it failed with `feel-editor` or something for older bundlers.